_includes/scripts.html

```diff
@@ -1,2 +1 @@
 <script src="{{ site.baseurl }}/assets/js/vendor/uswds.min.js"></script>
-<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
```

Comment:
Same comment as above in the `_includes/head.html` file, but you can restore this reference to `styleguide.js` here and remove the script tags for it in the `_includes/head.html` file in order to keep similar JS files grouped together.

Reply:
And done! Thank you!